.github/workflows/e2e_sft.yml

```diff
@@ -54,3 +54,7 @@ jobs:
         run: |
           ray stop --force
           bash tests/sft/run_sft_sp_loss_match.sh
+      - name: Running gsm8k e2e training tests on 8 L20 GPUs with sequence parallism and liger
+        run: |
+          ray stop --force
+          bash examples/sft/gsm8k/run_qwen_05_sp2_liger.sh 8 $HOME/ckpts/
```

Comment:
In the CI, can you upload a script that's only for the CI purpose in tests/sft?

The script should be identical to the example but only run one single step

Reply:
yep! fixed!